src/lhm.net/Entangler.cs

```diff
@@ -44,36 +48,36 @@ public IEnumerable<Func<string>> Entanglers
 
         private string CreateInsertTrigger()
         {
-            return string.Format(@"CREATE TRIGGER {0}_Insert ON [{0}] 
+            return string.Format(@"CREATE TRIGGER {0}_Insert_{1} ON [{0}] 
                                     AFTER INSERT 
                                     AS 
                                     BEGIN
-                                        SET IDENTITY_INSERT [{1}] ON
-                                        Insert into {1} ({2}) select {2} from inserted
-                                    END", _origin.Name, _destination.Name, _intersection.Insert);
+                                        SET IDENTITY_INSERT [{2}] ON
+                                        Insert into {2} ({3}) select {3} from inserted
+                                    END", _origin.Name, _timestamp, _destination.Name, _intersection.Insert);
```

Comment:
Just thinking how we clean this up I guess it can't be that hard to find all triggers on a table and clean it up.